### PR TITLE
Documentation for Starting a Thread and Installing from Hub

### PR DESCRIPTION
**Pull request by 0xgokuz**
Issue https://github.com/janhq/jan/issues/1133

Previous PR https://github.com/janhq/jan/pull/1437

Let me know what you think @hieu-jan @0xSage

### CODE REVIEW

File: docs/docs/guides/04-using-models/01-install-from-hub.mdx [hunk 1]
```diff
@@ -13,3 +13,44 @@ keywords:
     large language model,
   ]
 ---
+![Getting Started Highlight](assets/01-getting-started-install.gif)
+
+## How to use Jan
```
Comment (hieu-jan) [inline docs/docs/guides/04-using-models/01-install-from-hub.mdx:18]: Should we change into `Steps to Install Model From Hub`

Comment (0xgokuz) [reply docs/docs/guides/04-using-models/01-install-from-hub.mdx:18]: Do we want to call it Model or LLM? Thinking to stick with a single convention

File: docs/docs/guides/04-using-models/01-install-from-hub.mdx [hunk 2]
```diff
@@ -13,3 +13,44 @@ keywords:
...
+
+### 1. Search the LLM that you want to use from the Hub
```
[the remark this answers is not in the document]
Comment (0xgokuz) [reply docs/docs/guides/04-using-models/01-install-from-hub.mdx:20]: I thought about this, but then we'll need to replace all of the `LLM` keyword to `Large Language Models` seems too long?

In https://github.com/janhq/jan/pull/1444#discussion_r1444308962 I was also thinking whether we should call it Models vs. LLM

[the remark this answers is not in the document]
Comment (hieu-jan) [reply docs/docs/guides/04-using-models/01-install-from-hub.mdx:20]: I think we might go with `Large Language Models` for clear title.
Also having this idea, should we write a short introduction at the top of this guide. 
`In this guide, we will walk through the process of installing a Large Language Model (LLM) from the Hub.`. Then, we can use abbreviated form LLM for each content in description.

[the remark this answers is not in the document]
Comment (0xgokuz) [reply docs/docs/guides/04-using-models/01-install-from-hub.mdx:20]: Works. Let me update it.

**Comment by hieu-jan** [comment]
@0xgokuz Could you also help to separate this pull request into 2 different pull requests? Might be this one for `Install Models from the Hub` only. Then, I can merge your PR for `Install Models from the Hub` guide first. 

For `chatting/start-thread` guide, we will use `01-start-thread.mdx` and we might go to with this outline:

Rough outline:
- Choosing a model: Link to install model from hub
- Choosing an assistant
- Setting assistant instructions
     - At thread level
     - Globally, as default
- Customizing model params (thread level)
- Customizing engine params

File: docs/docs/guides/04-using-models/01-install-from-hub.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (hieu-jan) [reply docs/docs/guides/04-using-models/01-install-from-hub.mdx:22]: Thank you so much!!

**Pull request closed without merging**